.github/ISSUE_TEMPLATE/CONTENT.yml

```diff
@@ -0,0 +1,39 @@
+name: "Content Report 📄"
+description: Report an issue, a possible bug or anything that is confusing with Tauri docs.
+labels: ["improve documentation"]
+body:
+  - type: markdown
+    attributes:
+      value: "# Reminder: If you're reporting a minor correction, feel free to submit a PR directly!"
+  - type: input
+    id: subject
+    attributes:
+      label: 📚 Subject area/topic
+      description: Documentation area/topic (e.g. Authenticator, File System, CLI)
+      placeholder: "..."
+    validations:
+      required: true
```

Comment:
I think for me just having the page(s) is useful (that you have below), I don't think this would be that helpful tbh (and can be derived by which page they mentioned).